### PR TITLE
User's manual: no DVI, no Postscript, and assorted cleanups

### PR DESCRIPTION
**Pull request by xavierleroy**
This PR is best reviewed commit by commit.

The first commit removes the generation of the DVI and Postscript versions of the manual.  Those were fine formats in 1995 but nowadays they are superseded by PDF.

The second and third commit remove LaTeX hacks that have been useless since we use Hevea and the hyperref package.

### CODE REVIEW

File: manual/manual/tutorials/advexamples.etex [hunk 1]
```diff
@@ -17,8 +16,6 @@ programming pattern known as {\em virtual types} through the example
 of window managers.
 
 \section{Extended example: bank accounts}
-\pdfsection{Extended example: bank accounts}
-\label{ss:bank-accounts}
```
Comment (Octachron) [inline manual/manual/tutorials/advexamples.etex]: I think this label should not be deleted: it is referenced in section `3.4` .

Comment (gasche) [reply manual/manual/tutorials/advexamples.etex]: In fact, my intuition would be that having the labels cannot hurt, so we should/could also keep the other labels below (`ss:modules-as-classes`, etc.).

**Comment by xavierleroy** [comment]
Thanks for your keen eyes!  The removal of some labels was unintended, just an editing mistake on my side.  The labels are back.

File: manual/manual/plaintext.tex [hunk 2]
```diff
@@ -1,17 +0,0 @@
-\documentclass[11pt]{report}
-
-\usepackage{plaintext}
```
Comment (Octachron) [inline manual/manual/plaintext.tex:3]: I believe that `styles/plaintext.sty` should be removed along with this line.

**Review by Octachron — APPROVED** [review]
The changes look good to me, nevertheless I have a tiny remark on the plaintext cleanup.

**Comment by dra27** [comment]
Is it worth keeping the `pdfmanual` make target as an alias for the revised `manual` target?

**Comment by gasche** [comment]
(I personally don't care, I don't run the target often enough that it would be in muscle memory or anything.)

File: manual/manual/plaintext.tex (hunk 2 above)
Comment (xavierleroy) [reply manual/manual/plaintext.tex:3]: You're right, I removed it.

**Pull request merged**